pkgs/servers/xmpp/prosody/default.nix

```diff
@@ -80,6 +82,10 @@ stdenv.mkDerivation rec {
       wrapProgram $out/bin/prosody-migrator \
         --prefix LUA_PATH ';' "$LUA_PATH" \
         --prefix LUA_CPATH ';' "$LUA_CPATH"
+      ${lib.optionalString withOwnerAllowKickPatch ''
+        cat ${pkgs.jitsi-meet-prosody}/share/prosody-plugins/muc_owner_allow_kick.patch > $out/lib/prosody/modules/test
```

Comment:
what does this do? can't this `cat` just be omitted?

Reply:
thank you, it is ommited now